palm/plugins/dbt/commands/cmd_run.py

```diff
@@ -1,6 +1,7 @@
 import click
 from typing import Optional, Tuple
 from palm.plugins.dbt.dbt_palm_utils import shell_options, dbt_env_vars
+from palm.plugins.dbt.plugin_config import PluginConfig
```

Comment:
Unused import, this should be removed for now. (I know we have plans to use it in a follow-up PR)